VK_App/VK_App/CustomViews/CustomControlView.swift

```diff
@@ -77,13 +77,20 @@ final class CustomControl: UIControl {
     }
 
     private func setupViewContent() {
+        UIView.transition(
+            with: countLabel,
+            duration: 0.5,
+            options: .transitionFlipFromTop,
+            animations: { self.countLabel.text = "\(self.number)" },
+            completion: nil
+        )
         number = isSelected ? 1 : 0
         likeControlImageName = isSelected ? Constants.likeFillImageName : Constants.likeImageName
         countLabel.text = "\(number)"
         likeImageView.image = UIImage(systemName: likeControlImageName)
     }
 
-    @objc func likeTapAction(_ gesture: UITapGestureRecognizer) {
+    @objc private func likeTapAction(_ gesture: UITapGestureRecognizer) {
```

Comment:
```suggestion
    @objc private func likeTapAction() {
```
можно не передавать в параметры UITapGestureRecognizer если не используешь